spyderlib/start_app.py

```diff
@@ -95,6 +98,8 @@ def main():
             if TEST is None:
                 atexit.register(lock.unlock)
             from spyderlib import spyder
+            if report_time:
+                spyder.Stats.start_time = report_time

```

Comment:
It just saves the init start time. I couldn't do this from bootstrap, because `spyder` module is not imported there.